embassy-stm32/build.rs

```diff
@@ -1354,16 +1344,25 @@ fn main() {
     if &chip_name[..8] == "stm32wba" {
         println!("cargo:rustc-cfg={}", &chip_name[..8]); // stm32wba
         println!("cargo:rustc-cfg={}", &chip_name[..10]); // stm32wba52
+        println!("cargo:rustc-cfg=package_{}", &chip_name[10..11]);
+        println!("cargo:rustc-cfg=flash_{}", &chip_name[11..12]);
```

Comment:
this could be confused with the cfgs for the FLASH peripheral (like `flash_f4`), could you pick another name? maybe `mem` or `memory`? (I think that letter also affects RAM size?)

Reply:
could `size_` work? ST call it "flash memory size" typically. It sometimes influences SRAM too but not usually, so I think saying `mem` sounds a bit too much like it's talking about RAM when it's not. Could do `density` which is how ST used to refer to this on older devices.

Reply:
`size` sounds good. `density` please no :rofl:

Reply:
I've gone for `flashsize` for now, plain `size` seemed a big ambiguous.